Kickstarter-iOS/Features/Dashboard/Controller/DashboardViewController.swift

```diff
@@ -198,6 +203,43 @@ internal final class DashboardViewController: UITableViewController {
     }
   }
 
+  fileprivate func removeDeprecationWarning() {
```

Comment:
Using this function, we don't need a global variable in the view controller, we show and remove the view controller and any subviews every time the dashboard is shown and hidden. Limits the scope, basically a little cleaner.